test/verify/check-system-info

```diff
@@ -421,6 +430,85 @@ class TestSystemInfo(MachineCase):
 
         self.machine.execute("umount /sys/class/dmi")
 
+    @skipImage("Added in 190", "rhel-7-6-distropkg")
+    def testCPUSecurityMitigations(self):
+        b = self.browser
+        m = self.machine
+
+        self.login_and_go('/system/hwinfo')
+        b.wait_present('#hwinfo th:contains(CPU)')
+
+        def spoof_threads(threads_per_core, expect_link_present, expect_smt_state=None, cmdline=None):
+            if m.image in ['rhel-8-0']:
+                m.write('/usr/local/bin/lscpu', lscpu.format(threads_per_core, 3));
+            else:
+                m.write('/usr/local/bin/lscpu', lscpu.format(threads_per_core, ''));
+            m.execute('chmod +x /usr/local/bin/lscpu')
+            if cmdline:
+                m.write('/run/cmdline', cmdline)
+                m.execute('mount --bind /run/cmdline /proc/cmdline')
+
+            b.reload()
+            b.enter_page('/system/hwinfo')
+
+            if not expect_link_present:
+                b.wait_present('#hwinfo th:contains(CPU)')
+                b.wait_not_present('#hwinfo a:contains(Mitigations)')
+            else:
+                b.wait_present('#hwinfo a:contains(Mitigations)')
+                b.click('#hwinfo a:contains(Mitigations)')
+
+            if expect_smt_state:
+                b.wait_present('#cpu-mitigations-dialog span:contains(nosmt)')
+                b.wait_in_text('#cpu-mitigations-dialog #nosmt-switch .btn.active', expect_smt_state)
+
+            if cmdline:
+                m.execute('umount /proc/cmdline')
+                m.execute('rm /run/cmdline')
+
+        spoof_threads(1, False)
+        spoof_threads(2, True, 'On', 'param1 param2 nosmt param3=value3')
+        spoof_threads(2, True, 'On', 'param1 param2 nosmt=force param3=value3')
+        spoof_threads(2, False, cmdline='param1 nosmt=someunknown param3=value3')
+        spoof_threads(2, True, 'Off', None);
```

Comment:
Please add a comment "we expect our test VMs to not set nosmt". Also drop the stray semicolon.